Film Locations/Film Locations/MenuViewController.swift

```diff
@@ -62,17 +62,34 @@ class MenuViewController: UIViewController {
             
             // call before adding child view contoller's view as subview
             activeVC.didMove(toParentViewController: self)
-            activeViewController?.view.addSubview(toggleButton)
+            //activeViewController?.view.addSubview(toggleButton)
+            
+            setupToggleButton()
         }
     }
     
     private func setupToggleButton() {
-        toggleButton.frame = CGRect(x: 13, y: 18, width: 30, height: 30)
+        toggleButton.frame =  CGRect.zero
```

Comment:
Why do we have to execute the setupToogleButton every time a new ViewController is activated? I think it would be enough to do it once in ViewDidLoad and then to call other function setupToggleConstraints each time the button is added to the activeViewController. What do you think?